### PR TITLE
Segment docs

### PR DESCRIPTION
**Pull request by EG-tech**
as discussed in #314

### CODE REVIEW

File: Resources/Documentation/troubleshooting.md [hunk 1]
```diff
@@ -56,10 +56,6 @@ Error while decoding stream #0:0: Invalid data found when processing input
 
 You can safely ignore this warning, it's just FFmpeg complaining that it didn't receive a full frame of video when vrecord stopped.
 
-#### Sync Issues with FFmpeg 4.0 and qcli
```
Comment (dericed) [inline Resources/Documentation/troubleshooting.md]: There is a similar statement in settings.md, if removed, I suggest they be removed together.

File: Resources/Documentation/installation_and_setup.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
+## Installing vrecord
+
+If you use macOS, you can easily install vrecord using a package manager called Homebrew. To install Homebrew, follow [these instructions](https://brew.sh/).
```
Comment (retokromer) [inline Resources/Documentation/installation_and_setup.md]: >If you use macOS

It‘s not needed, as the classic `vrecord` is running only under macOS.

Comment (dericed) [reply Resources/Documentation/installation_and_setup.md]: i suggest this is not blocking, content changes could be in a subsequent pr

File: Resources/Documentation/troubleshooting.md (hunk 1 above)
Comment (EG-tech) [reply Resources/Documentation/troubleshooting.md]: Should a line instead be included to explain why the option exists? i.e., it is recommended to create *during* capture but the option to create after can be used to troubleshoot. I can see a user being confused as to how/why they would pick one over the other.

Comment (EG-tech) [reply Resources/Documentation/troubleshooting.md]: Also, the warning in settings.md was not mine, so I was trying to just get back to structural revision, dealing with content subsequently as you suggested

Comment (dericed) [reply Resources/Documentation/troubleshooting.md]: Replacing the warning with an explanation sounds fine to me, whether in this pr or a subsequent one.

Comment (EG-tech) [reply Resources/Documentation/troubleshooting.md]: I'll just remove the settings.md warning for now for consistency and to resolve #333 , can move feature clarification/further content audit into a subsequent PR

**Comment by dericed** [comment]
thx @EG-tech!!!!

**Pull request merged**